pages/seacreatures.jsx

```diff
@@ -55,63 +57,33 @@ export default function SeaCreaturesPage() {
     setShowDonated(!showDonated);
   }
 
-  const sortName = () => {
+  const sortByName = () => {
     setCrittersToShow(
-      [...crittersToShow].sort((critterA, critterB) => {
-        const nameA = critterA.name.toUpperCase();
-        const nameB = critterB.name.toUpperCase();
-        if (isAsc) {
-          if (nameA < nameB) {
-            return -1;
-          }
-          if (nameA > nameB) {
-            return 1;
-          }
-        } else {
-          if (nameA > nameB) {
-            return -1;
-          }
-          if (nameA < nameB) {
-            return 1;
-          }
-        }
-      })
+      [...crittersToShow].sort((critterA, critterB) =>
+        sortName(critterA.name, critterB.name, isAscAlph)
+      )
     );
-    setIsAsc(!isAsc);
+    setIsAscAlph(!isAscAlph);
   };
 
   const sortPrice = () => {
     setCrittersToShow(
-      [...crittersToShow].sort((critterA, critterB) => {
-        const priceA = parseInt(critterA.price);
-        const priceB = parseInt(critterB.price);
-        if (isAsc) {
-          if (priceA < priceB) {
-            return -1;
-          }
-          if (priceA > priceB) {
-            return 1;
-          }
-        } else {
-          if (priceA > priceB) {
-            return -1;
-          }
-          if (priceA < priceB) {
-            return 1;
-          }
-        }
-      })
+      [...crittersToShow].sort((critterA, critterB) =>
+        sortByNumbers(
+          parseInt(critterA.price),
+          parseInt(critterB.price),
+          isAscPr
+        )
+      )
     );
-    setIsAsc(!isAsc);
+    setIsAscPr(!isAscPr);
   };
 
-  const selectShadows = e => {
-    const shadow = e.target.value;
-    if (shadow === 'all') {
-      return setFilter({ ...filter, shadow: null });
-    }
-
-    setFilter({ ...filter, shadow });
+  const selectShadows = event => {
+    const shadow = event.target.value;
+    shadow === 'all'
+      ? setFilter({ ...filter, shadow: null })
+      : setFilter({ ...filter, shadow });
```

Comment:
If this is easier for you to understand, then leave it like this.
What I recommended was 
```suggestion
    setFilter({ ...filter, shadow: shadow === 'all' ? null : shadow });
```